### DISCUSSION TITLE
[Plugin idea] Native paydirekt

### DISCUSSION THREAD

**Discussion by Speedy1991** [pretix/pretix#1937] [Ideas]
Working with big german banks (e.g. VR, Sparkasse, Postbank, etc)
[https://www.paydirekt.de/](https://www.paydirekt.de/) scroll down

**Comment by Speedy1991** [comment]
API: [https://www.paydirekt.de/haendler/merchant-api.html](https://www.paydirekt.de/haendler/merchant-api.html)
Maybe a good starting point [https://github.com/ParticulateSolutions/django-paydirekt](https://github.com/ParticulateSolutions/django-paydirekt)

**Comment by raphaelm** [comment]
The trouble with paydirekt is, that it is quite cumbersome to get a merchant account. It is better than in the beginning (with the new concept of "merchant concentrators"), but I still don't expect many event organizers to go through the hassle.

That said, I'd love to see a pretix plugin anyway ;)
